baseline/services.py

```diff
@@ -799,7 +799,7 @@ def predict(self, tokens, K=1, **kwargs):
         examples = self.vectorize(tokens_batch)
 
         kwargs['beam'] = int(kwargs.get('beam', K))
-        outcomes = self.model.predict(examples, **kwargs)
+        outcomes, scores = self.model.predict(examples, **kwargs)
```

Comment:
I normally hate doing this but does it make sense to do like Keras and have a flag to decide what to return here?

Reply:
I guess that is an option, we are a bit inconsistent but the classifier has always returned classes and scores (on the other had the tagger has never returned scores)